atomics/T1059.004/T1059.004.yaml

```diff
@@ -185,6 +204,28 @@ atomic_tests:
       cat /etc/passwd |grep ^art
     cleanup_command: | 
       userdel art
+- name: Change login shell (freebsd)
+  description: |
+    An adversary may want to use a different login shell. The chsh command changes the user login shell. The following test, creates an art user with a /bin/sh shell, changes the users shell to sh, then deletes the art user. 
+  supported_platforms:
+    - freebsd
+  dependencies:
+  - description: |
+      chsh - change login shell, must be installed
+    prereq_command: |
+      if [ -f /usr/bin/chsh ]; then echo "exit 0"; else echo "exit 1"; exit 1; fi
+    get_prereq_command: |
```

Comment:
@alonsobsd I think validate [CI job is failing](https://github.com/redcanaryco/atomic-red-team/actions/runs/6726131477/job/18281835428?pr=2450#step:6:21) due to this being out of place, mind getting it to process a valid YAML, otherwise this looks great to me.

Reply:
It seems like some - linux and - freebsd entries were out of place. I committed some changes to see if it fix the issue. Thanks for your time

Reply:
Thanks for the quick reply @alonsobsd. Can you change line 72 in `bin/validate/validate.py` from 

```
commands=filter(lambda x: x is not None, commands)
```
to 
```
commands = list(filter(lambda x: x is not None, commands))
```

That should fix the issue in validate.py and you should get a verbose error. I dont have the rights to push to this PR or the branch for some reason.

Reply:
Looks like the test `Detecting pipe-to-shell (freebsd)` in T1059.004 doesn't have `executor` defined. `executor` is a required field.

Reply:
Done!